thoth/metrics_exporter/jobs/common.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+# thoth-metrics
+# Copyright(C) 2018, 2019 Christoph Görn, Francesco Murdaca, Fridolin Pokorny
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""A collection of methods that can be resued in different metric classes."""
+
+import logging
+
+from datetime import datetime
+from prometheus_api_client import PrometheusConnect
+import thoth.metrics_exporter.metrics as metrics
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def get_workflow_duration(
+    service_name: str,
+    prometheus: PrometheusConnect,
+    instance: str,
+    namespace: str,
+    check_time: datetime,
+    metric_type: metrics
+) -> datetime:
+    """Get the time spent for each worflow for a certain service."""
+    workflow_status_metric_name = "argo_workflow_status_phase"
+    workflow_status_metrics = prometheus.get_current_metric_value(
+        metric_name=workflow_status_metric_name,
+        label_config={
+            'instance': instance,
+            "namespace": namespace,
+            "phase": "Succeeded"}
+            )
+
+    if workflow_status_metrics:
+        new_time = datetime.now()
```

Comment:
Will be the use of `datetime.utcnow()` more suitable here?

Reply:
Yes you are right :) Thanks :candy: :cake: